src/@next/components/organisms/AddressGrid/AddressGrid.tsx

```diff
@@ -6,8 +6,10 @@ import { AddressTile } from "@components/molecules";
 import { IProps } from "./types";
 
 export const AddressGrid: React.FC<IProps> = ({ addresses }: IProps) => {
-  const addNewTile = [<AddNewTile type="address" />];
-  const addressTiles = addresses.map(address => <AddressTile {...address} />);
+  const addNewTile = [<AddNewTile key="0" type="address" />];
+  const addressTiles = addresses.map(address => (
```

Comment:
```
const addressTiles = addresses.reduce((elements, address) =>
 elements.push(<AddressTile key={address.id} {...address} />)
, addNewTile)
```
should be more efficient

Reply:
Added reduce but I had to modify this code a little bit as `push` returns length of new array and we must return array so that why code looks like this:

```
  const addressTiles = addresses.reduce(
    (elements, address) => {
      elements.push(<AddressTile key={address.id} {...address} />);
      return elements;
    },
    [addNewTile]
  );
```